code/datums/status_effects/debuffs.dm

```diff
@@ -968,3 +968,16 @@
 /datum/status_effect/cloudstruck/Destroy()
 	. = ..()
 	QDEL_NULL(mob_overlay)
+
+
+/datum/status_effect/ghoul
+	id = "ghoul"
+	status_type = STATUS_EFFECT_UNIQUE
+	duration = -1
+	examine_text = "<span class='warning'>SUBJECTPRONOUN has a blank, catatonic like stare.</span>"
```

Comment:
catatonic is used to to describe mobs without a client, so this might wording get confusing for players